travis/build-vm.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+TARGET="${TARGET:-/code/qemu.img}"
+RELEASE="${RELEASE:-stretch}"
+
+cd "$(dirname "$TARGET")"
+apt update
+apt -y install ./grml-debootstrap*.deb
+
+grml-debootstrap \
+  --force \
+  --vmfile \
+  --vmsize 3G \
+  --target "$TARGET" \
+  --bootappend "console=ttyS0,115200 console=tty0 vga=791" \
+  --password grml \
+  --release  "$RELEASE" \
+  --hostname "$RELEASE" \
+  # EOF
```

Comment:
the EOF line is a bit weird.

Reply:
Agreed, JFTR: it avoids having to keep the last line of actual code without `\`, so e.g. when adding a new cmdline option to the script or sorting lines later on then getting a possibly broken command line because of too much/too few `\` :))